packages/reakit/src/Tabbable/Tabbable.ts

```diff
@@ -195,8 +208,9 @@ export const useTabbable = createHook<TabbableOptions, TabbableHTMLProps>({
     return {
       ref: useForkRef(ref, htmlRef),
       style,
-      tabIndex: !trulyDisabled ? tabIndex : undefined,
-      disabled: trulyDisabled && nativeTabbable ? true : undefined,
+      tabIndex: !trulyDisabled ? tabIndex : !hasDisabled ? -1 : undefined,
```

Comment:
This is also adding `tabIndex={-1}` to elements that aren't natively tabbable (like `div`s), making them focusable. I think this is why tests are failing.

```suggestion
      tabIndex: !trulyDisabled ? tabIndex : nativeTabbable && !hasDisabled ? -1 : undefined,
```

Reply:
Sorry about that, I did not run the test but was only checking with the targetted examples.
I will improve this expression.
Thanks !!